src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusExporter.cs

```diff
@@ -63,6 +63,8 @@ public PrometheusExporter(PrometheusExporterOptions options)
 
     internal int ScrapeResponseCacheDurationMilliseconds { get; }
 
+    internal bool? OpenMetricsRequested { get; set; }
```

Comment:
Could we use a plain `bool` instead of a nullable one? This should be simple binary decision for the exporter.